content/wiki/capitalization_scheme.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Capitalization scheme for Linebender projects"
++++
+
+All Linebender projects should follow this capitalization scheme:
+
+- The project's name should start with an uppercase letter when referring to the project itself, eg Vello.
+- The name should be all-lowercase where referring to the project as a Rust crate (for instance, in relation to crates.io). In that case, it's encouraged to use backticks to put the project name in a codeblock, eg `vello`.
+- The name should be all-lowercase where referring to the project as a Github repository. In that case, it's encouraged to either use backticks or make it a link, eg [vello#1](https://github.com/linebender/vello/pull/1).
+
+Feel free to correct any documentation you see to follow this scheme.
+
+(Projects that are discontinued or in maintenance mode like Piet, Druid or Xi-Editor don't need their documentation to follow this scheme.)
```

Comment:
As Druid already follows this scheme I don't think it's needed to point it out as a project that doesn't need to follow it.